hook/iptables.go

```diff
@@ -21,6 +23,10 @@ const (
 	iptableHookForwardRuleTemplate        = "-d %s/32 -i !%s -o %s -p %s -j ACCEPT --dport %s"
 )
 
+// containerIpCache is a map containing the containers ips. This map is useful when
+// removing iptables when containers is stopped and does not have an IP anymore.
+var containerIPCache = sync.Map{}
```

Comment:
Please prefix by iptablesHook (iptablesHookContainerIPCache), since this variable is accessible from all hooks